packages/react-core/src/components/JumpLinks/JumpLinks.tsx

```diff
@@ -24,7 +24,12 @@ const getScrollItems = (children: React.ReactNode, hasScrollSpy: boolean) =>
   React.Children.toArray(children).map((child: any) => {
     if (hasScrollSpy && typeof document !== 'undefined' && child.type === JumpLinksItem) {
       const scrollNode = child.props.node || child.props.href;
-      if (typeof scrollNode === 'string' && typeof document !== 'undefined') {
+      if (typeof scrollNode === 'string') {
+        if (scrollNode.startsWith('#')) {
+          // Allow spaces and other special characters as `id`s
```

Comment:
Hmm? The link you have here mentions that spaces are not allowed to be used in IDs.

Reply:
Yes, but sadly Chrome allows them:
![image](https://user-images.githubusercontent.com/47335686/103544986-ea58f680-4e6e-11eb-9eca-3596e3e207a7.png)

I think using `docuement.getElementById(id)` instead of `document.getSelector('#' + id)` is more permissive and better.

Reply:
I think more or less every docs I could find was  clear about spaces not being allowed in IDs.

https://www.w3.org/TR/REC-html40/types.html#type-name or https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id 

Of course allowing them here does not break anything, so you can without any problem merge it, but it's slightly making the code more complicated and harder to read, with the benefit of offering the users the ability to add spaces in their hrefs, which is an undocumented behavior.

If it was up to me, I would not implement this corner case, to favor code simplicity and if any time we get a report about this not working as expected I would just reply with the above doc links.